.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 *.zip
 *.log
 .zapier-*
+.DS_Store
```

Comment:
fine to have here, but it's worth adding to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) (along with `.env` and stuff you'll never want in source control on purpose)